recipes/zmqpp/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class ZmqppConan(ConanFile):
+    name = "zmqpp"
+    homepage = "https://github.com/zeromq/zmqpp"
+    license = "MPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "This C++ binding for 0mq/zmq is a 'high-level' library that hides most of the c-style interface core 0mq provides."
+    topics = ("conan", "zmq", "0mq", "ZeroMQ", "message-queue", "asynchronous")
```

Comment:
they need to be lower case for searching 

```suggestion
    topics = ("conan", "zmq", "0mq", "zeromq", "message-queue", "asynchronous")
```